docs/kanvas-packages/workflows/rules.md

```diff
@@ -0,0 +1,124 @@
+# The Rule
+
+The rules are the main feature in this. The rule is compose for the Rule, the actions trigered and the conditions. Each Actions have the specifics System modules,
+For Example, PDF currently just work with the Messages system modules, no leads.
+
+## Before 
+Currently the Workflow no have the interface for create rule, here we going to create the rule in the database. You must have the system_modules_id of the entity 
+will to execute the rule, the companies_id.
+
+
+# Creating a new rule
+
+We going to create a rule  for this example we have a create a rule for sent the ADF and create the PDF. This rules will be trigered when a 
```

Comment:
"As an example of rule creation, down below a rule for sending an ADF and creating PDF will be provided."